### ISSUE TITLE
SQL error: near "AUTOINCREMENT": syntax error

### ISSUE THREAD

**Issue by Bigfootmech** [2shady4u/godot-sqlite#155]
Hey, I could be totally wrong here. 
New to this.

Trying to setup an "id" field (with datatype "int", primary key, autoincrement, not null, and unique

Except, if I make it "autoincrement" and "unique", Godot-SQLite fails with an SQL syntax error

eg
these work fine:
`	native_example["id"] = {"data_type":"int", "primary_key": true, "not_null": true, "auto_increment":true}`
`	native_example["id"] = {"data_type":"int", "primary_key": true, "not_null": true, "unique": true}`
this one will crash:
`	native_example["id"] = {"data_type":"int", "primary_key": true, "not_null": true, "auto_increment":true, "unique": true}`

Running a "query" with a create table statement made by "DB Browser for SQLite"

```
CREATE TABLE "Food_Simple" (
	"id"	INTEGER NOT NULL UNIQUE,
	"name"	TEXT NOT NULL UNIQUE,
	"calories"	REAL NOT NULL,
	PRIMARY KEY("id" AUTOINCREMENT)
);
```

also works just fine

So I'm assuming it's some internal logic to this extension? :S

**Comment by 2shady4u** [comment]
Hello @Bigfootmech,

What is the SQL syntax error?

**Comment by Bigfootmech** [comment]
Good question. Not entirely sure. How do I get Godot to show me a bigger stacktrace / more explanation?
![Screenshot_2906](https://github.com/2shady4u/godot-sqlite/assets/12558385/7fd88026-a8b5-48ee-a696-d6b36e2188eb)

**Comment by Bigfootmech** [comment]
I turned up the verbosity on the database, it seems that this statement is the one that fails

![Screenshot_2909](https://github.com/2shady4u/godot-sqlite/assets/12558385/86f1e69c-4e1a-4146-a4a2-7d05affe6a1d)

An online SQL validator also complains
https://www.eversql.com/sql-syntax-check-validator/

**Comment by 2shady4u** [comment]
Seems like this is an order problem:

Does not work:
`CREATE TABLE IF NOT EXISTS Food_Simple (id INTEGER PRIMARY KEY UNIQUE AUTOINCREMENT NOT NULL,name text NOT NULL);`

Works:
`CREATE TABLE IF NOT EXISTS Food_Simple (id INTEGER PRIMARY KEY AUTOINCREMENT UNIQUE NOT NULL,name text NOT NULL);`

I'll probably have to swap the order of `AUTOINCREMENT `and `UNIQUE `in the source code.
For now, as a work-around, you can do this:
`db.query("CREATE TABLE IF NOT EXISTS Food_Simple (id INTEGER PRIMARY KEY AUTOINCREMENT UNIQUE NOT NULL,name text NOT NULL);")`

(Instead of using the `create_table` convenience method)

**Comment by 2shady4u** [comment]
I have pushed a fix in c2bdbab

**Comment by 2shady4u** [comment]
Fix is implemented in the latest [release](https://github.com/2shady4u/godot-sqlite/releases/tag/v4.3)! (v4.3)
